feat(prover): add readme for guiding how to debug (#498)
taikoxyz/taiko-client: integration_test/README.md

> # how to debug test cases?

Reviewer: ```suggestion
# How to debug test cases?
```

* start docker compose
```
./docker/start.sh
```

* deploy L1 contracts
```

> # TAIKO_MONO_DIR variable is taiko-mono repo path.

Reviewer 2: ```suggestion
# replace $taiko-mono with the taiko-mono repo path.
```

TAIKO_MONO_DIR=$taiko-mono ./integration_test/deploy_l1_contract.sh
```

* show environment variables
```

> # TAIKO_MONO_DIR variable is taiko-mono repo path.

Reviewer 2: ```suggestion
# replace $taiko-mono with the taiko-mono repo path.
```

TAIKO_MONO_DIR=$taiko-mono ./integration_test/test_env.sh
```

> * copy that the result of previous step and release them in `Debug configurations`
> > after debugged don't forget stop docker compose

Reviewer 2: ```suggestion
* copy the result of previous step and paste it into `Debug configurations`
> after debugging, don't forget stop docker compose!
```
